source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -33,6 +33,8 @@ The following differences to the standard C++ range algorithms apply:
   In that case, the returned value contains iterators pointing to the positions past the last elements
   processed according to the algorithm semantics.
 - ``for_each`` does not return its function object.
+- ``reverse_copy`` returns ``std::ranges::in_in_out_result`` rather than its alias,
+  ``std::ranges::reverse_copy_truncated_result``.
```

Comment:
Depending on what should be considered as the "base" C++ standard, this wording might be inaccurate. As of now, the rest of the list compares with C++20/23. The new alias for the truncated result of parallel `reverse_copy` will only appear in C++26.

Reply:
I've updated that note in bb4ac307111891e6e2a1dc6126a1c7f80ee03a06 adding more details.